Packs/ExpanseV2/Scripts/ExpanseGenerateIssueMapWidgetScript/ExpanseGenerateIssueMapWidgetScript.py

```diff
@@ -21096,7 +21097,11 @@ def generate_map_command(args: Dict[str, Any]) -> str:
 
     annotations_image = Image.new('RGBA', (RESULT_IMAGE_X * 2, RESULT_IMAGE_Y * 2), (255, 0, 0, 0))
     draw = ImageDraw.Draw(annotations_image)
-    font = ImageFont.truetype("/usr/share/fonts/truetype/dejavu/DejaVuSansMono-Bold.ttf", 20)
+    if Path("/usr/share/fonts/truetype/dejavu/DejaVuSansMono-Bold.ttf"):  # debian location
```

Comment:
this should be in the rn too

Reply:
what should we write in the release-notes?

Reply:
Updated the script to support being run on a ubi9 based runner, or something like that

Reply:
removed